examples/protos/cluster.proto

```diff
@@ -0,0 +1,8 @@
+syntax = "proto3";
```

Comment:
why do you need these protos? you should use the ones from pitaya-protos

Reply:
Indeed, I will remove those and leave only `rpcans` and `rpcreq`, since both are "game protos"